validation.js

```diff
@@ -0,0 +1,14 @@
+const form = document.getElementById('form');
+const email = document.getElementById('email');
+const errorMessage = document.getElementById('error');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  if (email.value.toLowerCase() === email.value) {
+    form.submit();
+    errorMessage.style.visibility = 'hidden';
+  } else {
+    errorMessage.innerHTML = 'email address must be lower case';
+    errorMessage.style.visibility = 'visible';
+    errorMessage.style.color = 'green';
+  }
```

Comment:
[ OPTIONAL CHANGES ]

- [ ] It would be nice if you could make the error message be in red color or something easily visible as a warning to it stands out to the user that an error has occurred. 👍 
- [ ] Another suggestion to make the form fields look nice is to give them a little padding on the left so the text gets breathing space. See the screenshot below: I trust you can make them and they will help improve the user experience of your app 👍 

![image](https://user-images.githubusercontent.com/50941074/190637522-31ab3824-cd18-4456-b30c-67ac0bcc6d4b.png)